pkgs/mk-phony-oci/builder.sh

```diff
@@ -0,0 +1,81 @@
+set -eou pipefail
+. .attrs.sh
```

Comment:
where's this file?

Reply:
oh nevermind, it's a stdenv file

Reply:
It gets created via `__structuredAttrs = true;`